### ISSUE TITLE
Avoid instantiating Boolean objects; reference Boolean.TRUE or Boolean.FALSE or call Boolean.valueOf() instead.

### ISSUE THREAD

**Issue by carlwilson** [openpreserve/jhove#424]
There are seven occurrences of this issue.
Why is this an issue?
Since: PMD 1.2

Avoid instantiating Boolean objects; you can reference Boolean.TRUE, Boolean.FALSE, or call Boolean.valueOf() instead.

Example(s):
```java
Boolean bar = new Boolean("true");		// unnecessary creation, just reference Boolean.TRUE;
Boolean buz = Boolean.valueOf(false);	// ...., just reference Boolean.FALSE;
```
The full list of occurrences can be found [here on Codacy](https://app.codacy.com/app/openpreserve/jhove/issues?bid=3402463&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6W251bGxdfSx7ImlkIjoiQ2F0ZWdvcnkiLCJ2YWx1ZXMiOlsiRXJyb3JQcm9uZSJdfSx7ImlkIjoiTGV2ZWwiLCJ2YWx1ZXMiOltudWxsXX0seyJpZCI6IlBhdHRlcm4iLCJ2YWx1ZXMiOls5NjRdfSx7ImlkIjoiQXV0aG9yIiwidmFsdWVzIjpbbnVsbF19XQ==)

**Comment by carlwilson** [comment]
Hi @nvanderperren this is just a quick one really and not my final choice for you. I needed a quick choice as I have to drive to an office now. I suspect it won’t take long and I’ll be ready with another issue in a couple of hours.

**Comment by carlwilson** [comment]
Closed by #444
